docs/CHANGELOG.asciidoc

```diff
@@ -10,10 +10,496 @@
+
+*Impact* +
+Before you upgrade to 8.0.0., remove these settings from kibana.yml.
+====
+
+[discrete]
+[[breaking-113068]]
+.Adds requirement for inline scripting
+[%collapsible]
+====
+*Details* +
+To start {kib}, you must enable inline scripting in {es}. For more information, refer to {kibana-pull}113068[#113068].
+
+*Impact* +
+Enable link:https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-scripting-security.html[inline scripting].
+====
+
+[discrete]
+[[breaking-112773]]
+.Removes `kibana.index` settings
+[%collapsible]
+====
+*Details* +
+The `kibana.index`, `xpack.reporting.index`, and `xpack.task_manager.index` settings have been removed. For more information, refer to {kibana-pull}112773[#112773].
+
+*Impact* +
+Use spaces, cross-cluster replication, or cross-cluster search. To migrate to <<xpack-spaces,spaces>>, export your <<managing-saved-objects, saved objects>> from a tenant into the default tenant in a space. For more details, refer to link:https://github.com/elastic/kibana/issues/82020[#82020].
+====
+  
+[discrete]
+[[breaking-112305]]
+.Removes legacy logging
+[%collapsible]
+====
+*Details* +
+The logging configuration and log output format has changed. For more information, refer to {kibana-pull}112305[#112305].
+
+*Impact* +
+Use the new <<logging-configuration-migration,logging system>>. 
+====
+
+[float]
+[[reporting-breaking-changes-8.0.0-beta1]]
+==== Reporting
+      
+[discrete]
+[[breaking-114216]]
+.Removes reporting settings
+[%collapsible]
+====
+*Details* +
+The following settings have been removed: 
+
+* `xpack.reporting.capture.concurrency`
+
+* `xpack.reporting.capture.settleTime`
+
+* `xpack.reporting.capture.timeout`
+
+* `xpack.reporting.kibanaApp`
+
+For more information, refer to {kibana-pull}114216[#114216].
+
+*Impact* +
+Before you upgrade to 8.0.0, remove the settings from kibana.yml.
+====
+
+[float]
+[[rest-api-breaking-changes-8.0.0-beta1]]
+==== REST API 
+
+[discrete]
+[[breaking-114730]]
+.Removes `/api/settings`
+[%collapsible]
+====
+*Details* +
+The `/api/settings` REST API has been removed. For more information, refer to {kibana-pull}114730[#114730].
+
+*Impact* +
+Use `/api/stats`.
+====
+
+[float]
+[[security-breaking-changes-8.0.0-beta1]]
+==== Security
+
+[discrete]
+[[breaking-116191]]
+.Removes legacy audit logger
+[%collapsible]
+====
+*Details* +
+The legacy audit logger has been removed. For more information, refer to {kibana-pull}116191[#116191].
+
+*Impact* +
+To audit {kib} events, enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>.
+====
+
+[discrete]
+[[breaking-111681]]
+.Removes `xpack.security.enabled` setting
+[%collapsible]
+====
+*Details* +
+The `xpack.security.enabled` setting has been removed. For more information, refer to {kibana-pull}111681[#111681].
+
+*Impact* +
+Use {ref}/security-settings.html#general-security-settings[`xpack.security.enabled`] in elasticsearch.yml.
+====
```

Comment:
@jportner `Removes xpack.security.enabled setting` is documented in a separate section is because the work was completed in #111681, and `Removes enabled settings from plugins` was completed in #113495. I can combine them if needed.

Reply:
Yeah, we only completed it in a separate PR because we had some complex test changes to make as a result.

I'd personally lean towards a single section since that's more concise and the release notes are already massive.

Reply:
Done! Can you take a final look to make sure it still makes sense?

Reply:
Looks good